### PR TITLE
[PE-4557] Added Tooltip component page

### PR DESCRIPTION
**Pull request by janibolkvadze**
@mattnickles @jllr 

https://ctl.atlassian.net/browse/PE-4457

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://8eb8d55c59fcc9c85ccbf6e8c55afc0f6a9e0af9-chi-pr.enterprise.ctl.io" target="_blank"> https://8eb8d55c59fcc9c85ccbf6e8c55afc0f6a9e0af9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/704/ for more details.

File: src/website/views/components/tooltip/_examples.pug [hunk 1]
```diff
@@ -0,0 +1,51 @@
+h2 Base
+p.-text
+  | To render a tooltip, apply the <code>data-tooltip</code> attribute to an element. Then,
+  | you can initialize it with the chi.tooltip factory method.
+.example.-mb--3
+  .-p--3
+    button(data-tooltip='Your tooltip text on a button').a-btn.-mr--2 Tooltip
+  :code(lang='html')
+    <button id='tooltip-1' class="a-btn" data-tooltip="Your tooltip text on a button">Tooltip</button>
```
Comment (mattnickles) [inline src/website/views/components/tooltip/_examples.pug]:
```suggestion
    <button id="tooltip-1" class="a-btn" data-tooltip="Your tooltip text on a button">Tooltip</button>
```

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://cd4b0208caceac0fc5fe2a52042f520ad9120938-chi-pr.enterprise.ctl.io" target="_blank"> https://cd4b0208caceac0fc5fe2a52042f520ad9120938-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/714/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-t0ufxj, https://ops.elasticbox.com/services/instances/i-3n2fm3

**Pull request merged**